src/postgraphql/http/createPostGraphQLHttpRequestHandler.js

```diff
@@ -112,6 +119,58 @@ export default function createPostGraphQLHttpRequestHandler (options) {
       return
     }
 
+    // Serve the JavaScript for GraphiQL on a namespaced path
+    if (parseUrl(req).pathname === '/_postgraphql/graphiql.js') {
+      res.statusCode = 200
+      res.setHeader('Content-Type', 'text/javascript')
+      res.end(await graphiqlJs)
+      return
+    }
+
+    // Setup an event stream so we can broadcast events to clients
+    if (parseUrl(req).pathname === '/_postgraphql/stream') {
+      if (req.headers.accept !== 'text/event-stream') {
+        res.end()
+        return
+      }
+
+      // Making sure this options are set
+      req.socket.setTimeout(0)
+      req.socket.setNoDelay(true)
+      req.socket.setKeepAlive(true)
+
+      // Set headers for Server-Side Events
+      res.writeHead(200, {
+        'Content-Type': 'text/event-stream',
+        'Cache-Control': 'no-cache',
+        'Connection': 'keep-alive'
+      })
+
+      // Expose function on response to write to the stream
+      res.sse = string => {
```

Comment:
Yes. That will work. My thoughts were that you could also send sse further down the line if one wanted to. But will leave that to later when we actually need it.